js/interface.js

```diff
@@ -2226,7 +2368,20 @@ $('[data-save-rule]').click(function(event) {
     });
 
     rule.allow = { user: user };
-  } else {
+  } else if ($allow.data('allow') === 'tokens') {
+    selectedTokenId  = Number($('.tokens-list :selected').val());
+    var tokenFullName = _.find(integrationTokenList, function(token) {
+      return token.id === selectedTokenId;
+    });
+
+    if (tokenFullName) {
+      selectedTokenName = tokenFullName.fullName;
+    }
+
+    setSelectedTokenDetails(selectedTokenId, selectedTokenName);
+    rule.allow = {'tokens': [selectedTokenId]};
```

Comment:
Please explain why ESLint didn't catch this.

```suggestion
    rule.allow = { 'tokens': [selectedTokenId] };
```

Reply:
It's my bad. while checking the eslint issue with version 1.0.0 not getting but when I used 0.8.0 version it will found. I will take care this onwards.
![image](https://user-images.githubusercontent.com/102005879/190992096-bc5ef4c1-824b-46b2-a4cd-d2a91e3b8ac4.png)